lib/helpers/files.js

```diff
@@ -44,87 +44,37 @@ function packageLockJsonExists(cwd) {
 	return fileExists(path.join(cwd || process.cwd(), '/package-lock.json'));
 }
 
-function getBowerJson(cwd) {
-	cwd = cwd || process.cwd();
-	return requireIfExists(path.join(cwd, '/bower.json'));
-}
-
 function getOrigamiJson(cwd) {
 	cwd = cwd || process.cwd();
 	return requireIfExists(path.join(cwd, '/origami.json'));
 }
 
-function bowerJsonExists(cwd) {
-	return fileExists(path.join(cwd || process.cwd(), '/bower.json'));
-}
-
 function getMainSassPath(cwd) {
 	cwd = cwd || process.cwd();
 	const sassMainPath = path.join(cwd, '/main.scss');
-	let isInBowerMain = false;
-	return getBowerJson(cwd)
-		.then(bowerJson => {
-			if (bowerJson) {
-				if (Array.isArray(bowerJson.main) && bowerJson.main.includes('main.scss')) {
-					isInBowerMain = true;
-				} else if (typeof bowerJson.main === 'string' && bowerJson.main === 'main.scss') {
-					isInBowerMain = true;
-				}
-			}
-
-			return fileExists(sassMainPath);
-		})
-		.then(sassExists => {
-			if (isInBowerMain && !sassExists) {
-				// This should probably throw an error
-				// throw new Error('main.scss is listed in bower.json main, but file doesn\'t exist.');
-			} else if (!isInBowerMain && sassExists) {
-				// throw new Error('main.scss exists but is not listed in bower.json main.');
-			}
-			if (isInBowerMain && sassExists) {
-				return sassMainPath;
-			} else {
-				return null;
-			}
-		});
+	return Promise.resolve(sassMainPath);
 }
 
 function getMainJsPath(cwd) {
 	cwd = cwd || process.cwd();
-	const jsMainPath = path.join(cwd, '/main.js');
-	let isInBowerMain = false;
-	return getBowerJson(cwd)
-		.then(bowerJson => {
-			if (bowerJson) {
-				if (Array.isArray(bowerJson.main) && bowerJson.main.includes('main.js')) {
-					isInBowerMain = true;
-				} else if (typeof bowerJson.main === 'string' && bowerJson.main === 'main.js') {
-					isInBowerMain = true;
+	return getPackageJson(cwd)
+		.then(pkgJson => {
+			if (pkgJson) {
+				if (typeof pkgJson.main === 'string') {
+					return path.join(cwd, pkgJson.main);
+				} else if (typeof pkgJson.browser === 'string') {
+					return path.join(cwd, pkgJson.browser);
```

Comment:
i think we should prefer `browser` to `main`, because we're building for browsers

Reply:
Yes, let's do that 👍